proto/landing.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Google LLC
```

Comment:
naming nit: landing_page.proto to match the route?

Reply:
I am thinking putting all the landing page in this file, and have naming /landing/place(protein | disease). what do you think?

Reply:
yea, that sounds good. we can further split it up if it becomes too big..

Reply:
Yep, updated to /internal/place(protein) per discussion offline